src/toolbarActions/toolbarFunctions.js

```diff
@@ -80,15 +80,66 @@ const saveAction = (state) => {
 };
 
 async function saveGraphMLFile(state) {
+    const pickerOpts = {
+        types: [
+            {
+                description: 'Graphml',
+                accept: {
+                    'text/graphml': ['.graphml'],
+                },
```

Comment:
This particular method is to saveGraphML. While you may need to handle the programs elsewhere, the code block must not reside here.

Also, the bug is random and occasional. I don't think addition of this code block will fix the random occurrence of the bug.

Reply:
This is to remove browser warnings , 
I have seen your comment in that issue and now I will remove this thing

And please help me to merge this "Save" issue as i think It is working fine now as required